### PR TITLE
[stable12] allow admin to disable groups on personal page

### PR DESCRIPTION
**Pull request by schiessle**
backport of https://github.com/nextcloud/server/pull/5259

cc @MorrisJobke @nickvergessen @rullzer

### CODE REVIEW

**Comment by nickvergessen** [comment]
I prefer killing over config

**Review by MorrisJobke — APPROVED** [review]
Tested and works 👍  I would go for the config switch

**Comment by MorrisJobke** [comment]
Failed test is due to outdated autoloader - fixed in stable12

**Review by nickvergessen — APPROVED** [review]

**Comment by MorrisJobke** [comment]
Reverted in #5385

File: settings/personal.php [hunk 1]
```diff
@@ -160,6 +160,7 @@
 $uid = $user->getUID();
 $userData = $accountManager->getUser($user);
 
+$tmpl->assign('showGroupMembership', $config->getSystemValue('settings.personal.showGroupMembership', true));
```
Comment (Jessi1983) [inline settings/personal.php:163]: #

**Pull request merged**